modules/ROOT/pages/6.7-release-notes.adoc

```diff
@@ -287,6 +299,9 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === Applying an ordered or unordered list to a selected checklist incorrectly turned the list into paragraphs
 // TINY-9975
+Previously, when a xref:checklist.adoc[Checklist], particularly a checklist that contained a nested checklist, was selected and converted to a bulleted or numbered list, the selection was, incorrectly, converted into plain paragraphs.
```

Comment:
Converting a checklist to a numbered list worked fine. The bug only happens when trying to convert a checklist to a bulleted list
```suggestion
Previously, when a xref:checklist.adoc[Checklist], particularly a checklist that contained a nested checklist, was selected and converted to a bulleted list  the selection was, incorrectly, converted into plain paragraphs.
```